unity-renderer/Assets/DCLServices/ScreencaptureCamera/CameraObject/Scripts/ScreencaptureCameraBehaviour.cs

```diff
@@ -109,9 +110,12 @@ internal void Awake()
         {
             toggleScreenshotCameraAction = Resources.Load<InputAction_Trigger>("ToggleScreenshotCamera");
             toggleCameraReelAction = Resources.Load<InputAction_Trigger>("ToggleCameraReelSection");
+            exitScreenshotModeAction = Resources.Load<InputAction_Trigger>("CloseScreenshotCamera");
```

Comment:
do we plan on moving input actions to addressables?

Reply:
we can, but in general they don't take much of memory, so we will not gather any noticeable wins, just more consistency across the code base